scripts/examples/gn_silabs_example.sh

```diff
@@ -297,7 +297,7 @@ else
     fi
 
     # 917 exception. TODO find a more generic way
-    if [ "$SILABS_BOARD" == "BRD4338A" ]; then
+    if [ "$SILABS_BOARD" == "BRD4338A" ] || [ "$SILABS_BOARD" == "BRD2605A" ]; then
```

Comment:
This will not scale well with added WIFI soc boards and the above TODO is still true. Can it be integrated with some wifi_args.gni

Reply:
As `USE_WIFI `will be true for all wifi boards and based on it `build_for_wifi_gnfile.gn `is added for 917 SoC boards. Could you please suggest how to add soc boards in a generic way.